### PR TITLE
provide a toggle nullable feature when the user clicks on the same ra…

### PR DESCRIPTION
**Pull request by erangeles**
…ting again.

### CODE REVIEW

**Comment by erangeles** [comment]
@andrucz thoughts?

File: src/ionic2-rating.ts [hunk 1]
```diff
@@ -94,9 +94,13 @@ export class Ionic2Rating implements ControlValueAccessor {
   }
 
   rate(amount: number) {
-    if (!this.readOnly && amount >= 0 && amount <= this.range.length) {
-      this.value = amount;
+    if(number == this.innerValue) {
```
Comment (andrucz) [inline src/ionic2-rating.ts]: Do you mean `amount == this.innerValue`?

**Comment by andrucz** [comment]
That's an interesting feature. I am just wondering whether it should be activated by a property.

**Comment by andrucz** [comment]
I added ```nullable``` property (default value is false). Thanks.

**Pull request merged**